src/slic3r/GUI/Plater.cpp

```diff
@@ -5569,24 +5570,24 @@ void Plater::priv::reload_from_disk()
     for (auto [obj_idx, vol_idx] : selected_volumes) {
         const ModelObject *object = model.objects[obj_idx];
         const ModelVolume *volume = object->volumes[vol_idx];
-        if (fs::exists(volume->source.input_file))
-            input_paths.push_back(volume->source.input_file);
-        else {
-            // searches the source in the same folder containing the object
-            bool found = false;
-            if (!object->input_file.empty()) {
-                fs::path object_path = fs::path(object->input_file).remove_filename();
-                if (!object_path.empty()) {
-                    object_path /= fs::path(volume->source.input_file).filename();
-                    if (fs::exists(object_path)) {
-                        input_paths.push_back(object_path);
-                        found = true;
-                    }
+        // searches the source in the same folder containing the object
+        bool found = false;
+        if (!object->input_file.empty()) {
+            fs::path object_path = fs::path(object->input_file).remove_filename();
+            if (!object_path.empty()) {
+                object_path /= fs::path(volume->source.input_file).filename();
+                if (fs::exists(object_path)) {
+                    input_paths.push_back(object_path);
+                    found = true;
                 }
             }
-            if (!found)
-                missing_input_paths.push_back(volume->source.input_file);
         }
+        if (!found) {
```

Comment:
Can you explain a bit the changes here?

Reply:
If the file not found in the current directory and exists on the initial file path it will be reloaded from there. Otherwise user will be prompted to select file manually.

Reply:
> If the file not found in the current directory and exists on the initial file path it will be reloaded from there. Otherwise user will be prompted to select file manually.

This logic change sounds not very right to me.
The initial file path should have the priority IMO. It's also the logic in PS, can we revert the change?

Reply:
Sure. It's better to have Reload All with another logic in the main branch than not to have at all)
I will continue using this logic.

Reply:
This logic works the same way as original untill you copy amf project to another directory where new stl files with the same names exist. It's very convenient for me as a 3D designer when I have several versions of a model. You simply copy project to another folder, Reload All and you have new version ready to print.